edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -144,6 +144,11 @@
     <div class="inner-wrapper" style="color: black; overflow: auto;">
       <header><h2><span class="display_name">{{ display_name }}</span> - {% trans "Staff Graded Assignment" %}</h2></header>
       <br/>
+      <div>
+        <a class="instructor-info-action button btn-download-all" href="#" id="download-init-button">{% trans "Download All Submissions" %}</a>
+      </div>
+      <br/>
+      <div class="task-message"></div>
```

Comment:
(Sorry, forgot to mention this in the last round of comments) A few things that will make the UI a little bit cleaner:

1. Remove the `<br/>` at line 150
1. Make the `task-message` element a `<p>` instead of a `<div>`
1. Remove the `font-size` style rule for `.task-message` in the CSS

Here's the difference:

OLD
![ss 2017-12-13 at 12 26 01](https://user-images.githubusercontent.com/14932219/33952944-5b082c96-e001-11e7-9318-48eb04e75837.png)

NEW
![ss 2017-12-13 at 12 25 27](https://user-images.githubusercontent.com/14932219/33952952-5f908c90-e001-11e7-94bc-458be23d97b3.png)